### ISSUE TITLE
GSConnect is not working, it freezes the computer and the application crashes.

### ISSUE THREAD

**Issue by universish** [GSConnect/gnome-shell-extension-gsconnect#1870]
### Describe the bug

I activate it from the extensions app, then click on the button in the top right corner with the “ethernet-voice-off” logo. the drawer opens. I activate GSConnect, the computer starts to freeze. the application crashes. When I don't turn it off from extensions, the computer freezes and overclocking problem is not fixed. gnome v46. OS: fedora workstation 40

### GSConnect version

57

### Installed from

GNOME Extensions website

### GNOME Shell version

46.5

### Linux distribution/release

Fedora Workstation 40

### KDE Connect app version

24.08.0

### Plugin(s)

GSConnect gnome shell extension v57

**Comment by IAM-NEYE** [comment]
i don't think you need KDE connect is you are using GSConnect. Did you installed KDE Connect by yourself or it installed with GS connect( though i don't think it should be the case) 
**suggestion** : remove KDE connect and then enable gsconnect, if  it doesn't work try reinstalling GSconnect after trying  first suggestion.

**Comment by universish** [comment]
As far as I know GSConnect is just a GUI, maybe a GUI shortcut.  It uses kde connect as infrastructure.

**Comment by ferdnyc** [comment]
@universish No, sorry, it's exactly the opposite. You **must not** have `kdeconnect` running on a Linux system that uses GSConnect, as they will conflict with each other. GSConnect is a complete GNOME replacement for the `kdeconnect` Linux service. KDE Connect is only needed on the Android side, to communicate with GSConnect.

**Comment by ferdnyc** [comment]
(That's mentioned [in the Wiki install instructions](https://github.com/GSConnect/gnome-shell-extension-gsconnect/wiki/Installation#dependencies), and [on GSConnect's extensions.gnome.org page](https://extensions.gnome.org/extension/1319/gsconnect/).)

**Comment by ferdnyc** [comment]
I'm still not clear on why the conflict would be causing the freezing/crashing issues you mentioned, that's unexpected. Typically `kdeconnect` being active just means that GSConnect can't access the network ports it needs in order to communicate.

But I suppose it's possible that failing to bind the correct ports is causing the D-Bus service to be restarted in a loop, which could lead to resource exhaustion. It's probably worth investigating, but in terms of solving the immediate issue for you, deactivating/uninstalling `kdeconnect` should clear up the conflict and allow GSConnect to function normally.

**Comment by universish** [comment]
Today I deleted kde connect. I was going to install the GSConnect extension. The website server gave me an error: 

> 504 Gateway Time-out
> The server didn't respond in time. 

It was delayed until later.

**Comment by universish** [comment]
I also installed the GSConnect plugin today. No problem, it works fine. No cramping or freezing. 
The problem was having kde connect installed. Uninstall kde connect. install GSConnect from gnome-software store. If you don't have the GSConnect extension installed, install it from the Gnome extension web page or from the flathub “Extension Manager” application. The problem should be solved.

**Comment by ferdnyc** [comment]
Glad you got that sorted out, @universish!

I'm debating reopening this, to look into the issues with GSConnect making GNOME Shell / the system itself unstable, if used with kdeconnect installed... but I'm not sure there's that much likelihood of anyone doing much to address that issue, and really it's best to just avoid the situation. This can probably stay closed for now, we might need to revisit it if the issue comes up again in the future.

**Comment by dmzoneill** [comment]
I tried this twice, on fedora 42/43 and gnome 48/49.  
The laptop hard freezes (lenovo p1 gen7 2025) after a period of time.
I consider this software a virus.  
it's litterly that bad; a total denial of service extension.  
Whole machine hard locks up. 
The only option is the power button.

i considered reporting the extension, but ill leave it at that.
